src/status_im/chat/events/send_message.cljs

```diff
@@ -1,102 +1,58 @@
 (ns status-im.chat.events.send-message
   (:require [re-frame.core :as re-frame]
-            [status-im.utils.handlers :as handlers]
+            [status-im.chat.utils :as chat-utils]
+            [status-im.chat.models.message :as message-model]
             [status-im.constants :as constants]
-            [status-im.utils.clocks :as clocks]
-            [status-im.utils.config :as config]
-            [status-im.chat.models :as chat.models]
-            [status-im.chat.utils :as chat.utils]
-            [status-im.protocol.core :as protocol]
+            [status-im.data-store.chats :as chats-store]
             [status-im.native-module.core :as status]
+            [status-im.protocol.core :as protocol]
+            [status-im.utils.config :as config]
+            [status-im.utils.handlers :as handlers]
+            [status-im.utils.random :as random]
+            [status-im.utils.types :as types]
+            [status-im.utils.datetime :as datetime]
             [taoensso.timbre :as log]))
 
 (re-frame/reg-fx
-  ::send-notification
-  (fn [fcm-token]
-    (log/debug "send-notification fcm-token: " fcm-token) 
+  :send-notification
+  (fn [[fcm-token message]]
```

Comment:
Why we pass `[fcm-token message]` vector here when `message` is not used anywhere in the function ?